sdcm/utils/common.py

```diff
@@ -1560,7 +1615,9 @@ def get_branched_gce_images(scylla_version: str, project: str = SCYLLA_GCE_IMAGE
         reverse=True,
     )
 
-    assert images, f"GCE images for {scylla_version=} not found"
+    if not images:
```

Comment:
keep the assert, catch it on the cli function if you want different output

Reply:
Fixed (reverted to an assert and handled in the calling function)